core/domain/stats_jobs_one_off.py

```diff
@@ -436,10 +431,9 @@ def prepare_reduce(cls, exp_id, values):
         for value in values:
             if value['version'] is None:
                 error_messages.append(
-                    'None version for %s %s %s %s %s' % (
+                    'None version for %s %s %s' % (
                         exp_id,
-                        value['event_type'], value['state_name'], value['id'],
-                        value['created_on']))
+                        value['event_type'], value['state_name']))
```

Comment:
`prepare_map` function does not return 'id' and 'created_on', that's why I removed those fields. See here for more info:
https://github.com/oppia/oppia/blob/574b643cf8f1dea6e93d1471c57a019af32cbfcb/core/domain/stats_jobs_one_off.py#L314

Reply:
Done